components/schemas/billing/Service.yml

```diff
@@ -28,7 +28,7 @@ properties:
   item:
     "$ref": "./ServiceItem.yml"
   discount:
-    "$ref": "./Discount.yml"
+    "$ref": "./discounts/Discount.yml"
```

Comment:
Please update this to match the name in the platform - `AssociatedDiscount` I believe.